SofascoreProject/SofascoreProject/View/LeagueView.swift

```diff
@@ -0,0 +1,39 @@
+//
+//  LeagueView.swift
+//  SofascoreProject
+//
+//  Created by Sven Leko on 15.03.2024..
+//
+
+import Foundation
+import SofaAcademic
+import SnapKit
+import UIKit
+
+public class LeagueView: BaseView {
+    var logo: UIImageView = .init()
+    var countryAndNameHolder : CountryAndNameHolder = .init()
+    
+    
+    override public func addViews() {
+        addSubview(logo)
+        addSubview(countryAndNameHolder)
+    }
+    
+    override public func setupConstraints() {
+        logo.snp.makeConstraints {
+            $0.top.equalToSuperview().inset(12)
+            $0.leading.equalToSuperview().inset(16)
+            $0.width.height.equalTo(32)
+        }
+        
+        countryAndNameHolder.snp.makeConstraints {
+            $0.top.equalToSuperview().inset(16)
+            $0.leading.equalToSuperview().inset(80)
+            $0.bottom.equalToSuperview().inset(16)
+            $0.width.equalToSuperview().offset(-129)
```

Comment:
Generalno ako imas parent/child odnos tj. `equalToSuperview()` koristis `inset()`, a ak trebas constrainat dva "bratska" viewa koritis `offset`, primjer: https://velog.io/@juh2/SnapKit-offset-inset
```suggestion
            $0.width.equalToSuperview().inset(129)
```